.github/workflows/spanner-terraform-validator.yml

```diff
@@ -11,15 +12,14 @@ on:
     # Include spanner terraform sample paths only
     - '.github/workflows/spanner-terraform-validator.yml'
     - 'v2/datastream-to-spanner/terraform/samples/**'
-    - 'v2/sourcedb-to-spanner/terraform/samples/**'
 
 jobs:
   verify-terraform-samples:
     runs-on: ubuntu-latest
     strategy:
       matrix:
         terraform_paths:
-        - v2/sourcedb-to-spanner/terraform/samples
+        - v2/datastream-to-spanner/terraform/samples
```

Comment:
is this expected ? It seems a little arbitrary to change

Reply:
I added a number of additional validations so I've excluded the terraform example for bulk from here. That sample would be updated in a separate PR and then we can enable the validator check on it.